lambdatimes/src/components/Content/Content.js

```diff
@@ -48,9 +58,14 @@ export default class Content extends Component {
           `selectedTab` that includes the currently selected tab
           and `selectTabHandler` that includes the function to change the selected tab
         */}
-        <Tabs tabs={this.state.tabs} />
+        <Tabs tabs={this.state.tabs}
+          selectedTab={this.state.tabs} 
```

Comment:
Should be set to `this.state.selected`. `tabs` points to the entire array in the state. Making this change will enable your active tab style to work.

Reply:
Sorry when I do {this.state.selected} it wont let put info into the password field. I took it off to do CSS.

Reply:
NM I was thinking about a different project when I replied